.github/workflows/release.yml

```diff
@@ -19,10 +23,19 @@ jobs:
       - uses: actions/setup-go@c4a742cab115ed795e34d4513e2cf7d472deb55f # tag=v3
         with:
           check-latest: true
+      - name: Generate goreleaser args
+        id: args
+        run: |
+          set -euo pipefail
+          args='release --rm-dist'
+          if [[ "$GITHUB_REF" != refs/tags/* ]]; then
+            args+=' --snapshot'
+          fi
+          echo "args=$args" >> $GITHUB_OUTPUT
```

Comment:
Thank you for using the new format. Would you mind creating a separate PR to update all our `set-output` instances the same way? It would be greatly appreciated ❤️.